app.py

```diff
@@ -20,87 +16,66 @@ def on_run_app():
         beers = {}
 
 
-def on_exit_app():
+@app.on_event("shutdown")
+def shutdown_event():
     with open("data.json", "w+") as outfile:
         json.dump(beers, outfile)
 
 
-atexit.register(on_exit_app)
+@app.get("/beers/{isbn}")
+async def get_beer(isbn: str):
+    if isbn in beers:
+        return beers[isbn]
+    raise HTTPException(status_code=404, detail="Item not found")
```

Comment:
Be more "nice" to your API users, give more details in the error message
```suggestion
    raise HTTPException(status_code=404, detail=f"Beer {isbn} not found")
```